src/composables/resetAccountWithFirebase.js

```diff
@@ -0,0 +1,18 @@
+import { updatePassword } from "firebase/auth";
+import { appAuth } from "./firebaseConfig/config";
+
+export function resetUserAccount(email, newPassword) {
+    const auth = appAuth();
+
+    auth.sendPasswordResetEmail(email)
```

Comment:
`sendPasswordResetEmail` is not a method to auth... Use the documentation to see appAuth should be a parameter just like email